### PR TITLE
Peerio app

### PR DESCRIPTION
**Pull request by 0xadada**
Add Peerio.app :latest

Latest version is currently 1.0.2
Peerio is an encrypted chat and file sharing utility.

### CODE REVIEW

File: Casks/peerio.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'peerio' do
+  version :latest
+  sha256 :no_check
+
+  url "https://peerio.com/download/peerio-mac.zip"
+  name 'Peerio'
+  homepage 'https://peerio.com/'
+  license :oss
```
Comment (vitorgalvao) [inline Casks/peerio.rb]: `:gpl`

**Comment by vitorgalvao** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git).

**Comment by ronaldjadams** [comment]
Pushed the GPL changes, ran rebase, but dunno if it worked.

**Comment by ndr-qef** [comment]
Not quite, unfortunately. @ronaldjadams, thanks for adding this new cask. I merged your PR as 3c929e5.

**Pull request closed without merging**